website/docs/d/prefix_list.html.markdown

```diff
@@ -53,6 +64,9 @@ whose data will be exported as attributes.
 * `prefix_list_id` - (Optional) The ID of the prefix list to select.
 
 * `name` - (Optional) The name of the prefix list to select.
+* `filter` - (Optional) One or more name/value pairs to use as filters. There are
+several valid keys, for a full reference, check out
+[describe-prefix-lists in the AWS CLI reference][1].
```

Comment:
Can you please update this documentation to the longer form configuration block style? e.g. https://github.com/terraform-providers/terraform-provider-aws/pull/12400/files#diff-259bbf21af9ff942736c706b725f4d6dR76-R88 Thank you!